webui/src/components/OKR.vue

```diff
@@ -109,6 +115,24 @@
                 this.logChange(`Me deleted '${objId}'`);
             },
 
+            sendChangeSuggestion(objective) {
+                const targetSubject = this.$store.state.user.selectedSubject;
+                window.location = `mailto:${targetSubject.mail || targetSubject.userPrincipalName}?\
+subject=Objective: ${objective.statement}&\
+body=Hi ${targetSubject.givenName || ''}. 
```

Comment:
Cannot understand how to make body multi line.
Probably can do it later when will work on correct text.